iree/testing/status_matchers.h

```diff
@@ -304,31 +324,42 @@ inline internal::IsOkMatcherGenerator IsOk() {
 }  // namespace status
 }  // namespace testing
 
+// TODO(#265): rename to IREE_*_OK.
 // Macros for testing the results of functions that return iree::Status or
 // iree::StatusOr<T> (for any type T).
-#define EXPECT_OK(rexpr) EXPECT_THAT(rexpr, ::iree::testing::status::IsOk())
-#define ASSERT_OK(rexpr) ASSERT_THAT(rexpr, ::iree::testing::status::IsOk())
+#define IREE_EXPECT_OK(rexpr) \
+  EXPECT_THAT(rexpr, ::iree::testing::status::IsOk())
+#define IREE_ASSERT_OK(rexpr) \
+  ASSERT_THAT(rexpr, ::iree::testing::status::IsOk())
+
+// #define IREE_ASSERT_OK(expr) \
+//   ASSERT_EQ(IREE_STATUS_OK, iree_status_consume_code(expr))
+// #define IREE_EXPECT_OK(expr) \
+//   EXPECT_EQ(IREE_STATUS_OK, iree_status_consume_code(expr))
```

Comment:
Remove? Or do you want to keep the `iree_status_consume_code` usage / enable later?